components/automate-ui/src/app/pages/integrations/integrations-form/integrations-form.component.html

```diff
@@ -33,7 +33,8 @@
 
     <header>
       <h3 class="display5">Specify which credentials to be used with tag key value (optional)</h3>
-      <span class="help-text">Should you wish to run a scan job on these instances, you can associate ssh and winrm creds with instances by tags. Wild card match(*) allowed in value.</span>
+      <span class="help-text">Should you wish to run a scan job on these instances, you can associate ssh and winrm
```

Comment:
@vjeffrey Would this span being two lines long, rather than one, cause issues?

Reply:
oh i can switch it back to one. that was my editor formatting things

Reply:
@vjeffrey I don't have any further comments/edits other than that. :)